src/main/java/it/gov/pagopa/pu/bff/connector/debt_position/DebtPositionService.java

```diff
@@ -4,14 +4,17 @@
 import it.gov.pagopa.pu.debtpositions.dto.generated.DebtPositionDTO;
 import it.gov.pagopa.pu.debtpositions.dto.generated.PagedModelDebtPosition;
 import it.gov.pagopa.pu.debtpositions.dto.generated.PagedModelDebtPositionView;
-import java.util.List;
 import org.springframework.data.domain.Pageable;
 
+import java.util.List;
+
 public interface DebtPositionService {
   DebtPositionDTO createDebtPosition(DebtPositionDTO debtPositionDTO, boolean massive, String accessToken);
 
   PagedModelDebtPositionView getDebtPositionViews(DebtPositionViewFiltersDTO filtersDTO, List<String> debtPositionOrigins, String operatorExternalId, Pageable pageable, String accessToken);
 
   DebtPositionDTO getDebtPosition(Long debtPositionId, String accessToken);
   PagedModelDebtPosition getDebtPositionByDebtPositionTypeOrgId(Long debtPositionTypeOrgId, Pageable pageable, String accessToken);
+
+  Boolean deleteDebtPosition(Long debtPositionId, String accessToken);
```

Comment:
let's add a javadoc to describe what's the meaning of true or false
```suggestion
  boolean deleteDebtPosition(Long debtPositionId, String accessToken);
```